app/jobs/spree_mailchimp_ecommerce/create_store_job.rb

```diff
@@ -1,14 +1,15 @@
 module SpreeMailchimpEcommerce
   class CreateStoreJob < ApplicationJob
     def perform(*_args)
-      ::Gibbon::Request.new(api_key: mailchimp_api_key).
-        ecommerce.stores.create(body: {
-                                  id: mailchimp_store_id,
-                                  list_id: mailchimp_list_id,
-                                  name: mailchimp_store_name,
-                                  currency_code: "USD",
-                                  domain: ENV['BASE_URL']
-                                })
+      response = ::Gibbon::Request.new(api_key: mailchimp_api_key).
+                 ecommerce.stores.create(body: {
+                                           id: mailchimp_store_id,
+                                           list_id: mailchimp_list_id,
+                                           name: mailchimp_store_name,
+                                           currency_code: "USD",
```

Comment:
@chhga let's not hardcode `USD`, let's use `Spree::Store.default.default_currency` or `Spree::Config[:currency']`